### ISSUE TITLE
Apply to Docker Open Source Program

### ISSUE THREAD

**Issue by lemeurherve** [jenkins-infra/helpdesk#2842]
https://www.docker.com/blog/expanded-support-for-open-source-software-projects/

https://www.docker.com/community/open-source/application/

> Docker is committed to supporting the Open Source community. The Docker Open Source Program provides a platform to help non-commercial open source developers continue to collaborate, innovate and push their projects forward.
> 
> For your project namespace on Docker Hub, Docker offers unlimited pulls and unlimited egress to any and all users, with no egress restrictions applying to any Docker users pulling images from that namespace. In addition, if your project uses Autobuild capabilities, you can continue using them for free. You are also free to continue to use Docker Desktop via the Docker Personal subscription. 
> 
> Qualification Criteria
> 
> To qualify for the program, your project namespace:
> 
> -  Must be shared in public repos.
> -  Is not funded by a commercial company or organization.
> -  Meets the Open Source Initiative definition (defined here), including definitions for Free distribution, source code, derived works, integrity of source code, licensing and no tolerance for discrimination Distributes images under an OSI approved open source license.

Looks good

_Originally posted by @lemeurherve in https://github.com/jenkins-infra/helpdesk/issues/2837#issuecomment-1075102619_

**Comment by timja** [comment]
I thought we already had this, cc @olblak

**Comment by olblak** [comment]
yes we are in

**Comment by dduportal** [comment]
We are, but which account ? Jenkinsciinfra ? (Because i’m almost sure that jenkins4eval i ms not)

**Comment by olblak** [comment]
I am forwarding the email thread to @dduportal and @lemeurherve

**Comment by timja** [comment]
just the jenkins org I think

**Comment by dduportal** [comment]
Additional point: I assume that any "open source subscription" allow to generate DockerHub APi token with the read-only mode, is that correct?

**Comment by timja** [comment]
no idea

**Comment by olblak** [comment]
And when I asked, They told me that we couldn't see it on our side

**Comment by olblak** [comment]
All we see is from the default free plan

**Comment by dduportal** [comment]
- Application form filled for the `jenkinsciinfra` and `jenkins4eval` org (it did not exist back in 2020) to provide a "normal process" and "numbers" to Docker Inc.
- Tried to contact the person that @olblak exchanged with in Nov. 2020 but this person does not work for Docker anymore (cced the private mailing list jenkins-infra-team)
- Asked someone at Docker that I know if this is the "standard" procedure

**Comment by dduportal** [comment]
For information, found some key issues on the DockerHub public roadmap:

- https://github.com/docker/roadmap/issues/20#issuecomment-1098929099
- https://github.com/docker/roadmap/issues/115
- https://github.com/docker/hub-tool/issues/172
- https://github.com/docker/roadmap/issues/165

**Comment by timja** [comment]
`jenkins4eval` has been around a long time and is not used much

**Comment by dduportal** [comment]
> `jenkins4eval` has been around a long time (i.e. pre 2020) and is not used much

Yes, but if under the open source program umbrella, it would be the perfect org. to provide read-only token for different users (to be used for docker pulls), avoiding to create addition accounts

**Comment by dduportal** [comment]
OK, good news, I was able to get another contact at Docker Inc.. Email sent with the jenkins-infra-team (private) mailing list in Cc.

**Comment by dduportal** [comment]
Update: 
We had email exchanges + an interview with Sharon Chang and Britney Blodget at Docker Inc..

- The organizations `jenkins4eval` and `jenkinsciinfra` are accepted to be part of the Open Source Program
- ETA undefined, as Docker Inc. is going through  a major conference (DockerCon) + program revamp for Open Source, so it slows down their ability to add us, but it's ongoing and should happen soon
- They are REALLY interested in a blog post about "BuilldX Bake".

**Comment by dduportal** [comment]
Has this task is "blocked" by external actor, gotta move it to the "infra-team-sync-next" milestone (e.g. top of the stack) to keep an eye on it.
